cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst

```diff
@@ -147,5 +147,5 @@ If there are any errors compiling ``.po`` files run the following command to val
     $ make validate
 
 See `django's i18n troubleshooting documentation
-<https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#troubleshooting-ugettext-incorrectly-detects-python-format-in-strings-with-percent-signs>`_
+<https://docs.djangoproject.com/en/2.2/topics/i18n/translation/#troubleshooting-gettext-incorrectly-detects-python-format-in-strings-with-percent-signs>`_
```

Comment:
I don't know if this is the equivalent link, since I don't know the details of ugettext vs gettext.

Reply:
This is the correct link.  ugettext was relevant in python 2 because we wanted unicode text and that was not the default.   In python 3 gettext should be correct.